yrs/src/transaction.rs

```diff
@@ -578,11 +567,11 @@ impl<'a> Transaction<'a> {
         }
     }
 
-    pub(crate) fn add_changed_type(&mut self, parent: &Branch, parent_sub: Option<&String>) {
-        let trigger = match parent.item.as_ref() {
-            None => true,
-            Some(ptr) if ptr.id.clock < (self.before_state.get(&ptr.id.client)) => {
-                if let Some(item) = self.store.blocks.get_item(ptr) {
+    pub(crate) fn add_changed_type(&mut self, parent: &Branch, parent_sub: Option<Rc<str>>) {
+        let trigger = match &parent.ptr {
```

Comment:
`Branch::item` has been removed - we can fully describe equivalent functionality using already existing `Branch::ptr` field.